src/index.html

```diff
@@ -1,13 +1,490 @@
+                  Explore
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact" id="Contacts">
+        <div class="container">
+          <h2 class="page__title">
+            Contact Us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="
+              grid__item--tablet-1-3
+              grid__item--desktop-1-5"
+            >
+              <form
+                class="form"
+                action="#"
+                method="get"
+              >
+                <input
+                  type="text"
+                  name="Name"
+                  placeholder="Name"
+                  class="contact__form"
+                  required
+                >
+
+                <input
+                  type="email"
+                  name="Email"
+                  placeholder="Email"
+                  class="contact__form"
+                  required
+                >
+
+                <textarea
+                  name="Message"
+                  placeholder="Message"
+                  class="
+                    contact__form
+                    contact__form--textarea
+                    contact__form--last"
+                  required
+                ></textarea>
+
+                <div class="
+                  grid__item--tablet-1-3
+                  grid__item--desktop-1-5"
+                >
+                  <button class="button button--primary" id="formButton">
+                    Send
+                  </button>
+                </div>
+              </form>
+            </div>
+
+            <div class="
+              grid__item--tablet-4-6
+              grid__item--desktop-8-11"
+            >
+              <div class="contact__info">
+                <h4 class="contact__info-title">
+                  Phone
+                </h4>
+
+                <a
+                  href="tel: +1 234 5555-55-55" class="contact__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact__info">
+                <h4 class="contact__info-title">
+                  Email
+                </h4>
+
+                <a
+                  href="mail: hello@miami.com" class="contact__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+
+              <div class="contact__info">
+                <h4 class="contact__info-title">
+                  Address
+                </h4>
+
+                <a
+                  href="https://goo.gl/maps/ZJjBqD3pwGUp8PSP7"
+                  class="contact__info-link"
+                  target="_blank"
+                >
+                  400 first ave.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
```

Comment:
br tag not working))
![image](https://user-images.githubusercontent.com/98812993/202847291-6718db91-d0f1-4806-8c66-641a783f06aa.png)

Reply:
and a link doesn't open location in the new tab